src/Config/upiconfig.xml

```diff
@@ -14,7 +14,7 @@
             <upi_attribute name='id' value='Win64'/>
             <upi_element name='level'>
               <upi_attribute name='name' value='build'/>
-              <upi_attribute name='id' value='3.3.0' />
+              <upi_attribute name='id' value='3.3.1' />
```

Comment:
This remains 3.3.0 a few [line](https://github.com/DynamoDS/Dynamo/pull/15818/files#diff-5d575576149eddb2a8d9156699f04769b3f5f89e6bea5e47e36224b7986e63dbR11) above for release?

Reply:
Looking at other point releases that seems to be the case.